luxonis_train/nodes/backbones/ddrnet/ddrnet.py

```diff
@@ -304,3 +287,28 @@ def init_params(self) -> None:
             elif isinstance(m, nn.BatchNorm2d):
                 nn.init.constant_(m.weight, 1)
                 nn.init.constant_(m.bias, 0)
+
+    @override
+    def get_weights_url(self) -> str:
```

Comment:
@klemen1999 I noticed from the coverage report that we're not actually using the pretrained weights for the `DDRNet` backbone in the predefined segmentation model. We're only using them in the head. I feel like this is probably a mistake? Or do you know whether that was intentional?